samples/powerpoint/basics/insert-image.yaml

```diff
@@ -27,7 +27,7 @@ script:
4A4fQsak862Lxd4ESWrR70kmt2BginZ1GjgxjAmtWJHvOf2RkgnJ5FjQ5iAAtWJ3rMf2ZngHB6FjU6iAEsWP3oPdNBfKVnUfuug82rBU5nwern8ajtDBDu8So+RgcxhtWqn8ejtjNAuMer+BgdxBhWq662R21ngHCP1H2MDmIMS1VvdgYIp9bVmg427xXoxFLVlZ0Bwil3NTqIYSxVXdkZIJxyV6ODGMZS1ZWdAcIpdzU6iGEsVV3ZGSCcclejgxjGUtWVnQHCKXc1OoiRrFb92BkgnHJX+9rB5pkCXVmt+rEzQDgVr0YHMZLVqh87A4RT8Wp0ECNZrfqxM0A4Fa9GBzGS1aofOwOEU/FqdBAjWa36sTNAOBWvRgcxktWqHzsDhFPxanQQI1mt+rEzQDgVr0YHMZgFqxM7A4RT8WqfOtg8UOACFqxO7AwQTtGr0UEMZsHqxM4A4RS9Gh3EYBasTuwMEE7Rq9FBDGbB6sTOAOEUvRodxGAWrE7sDBBO0avRQQxmwerEzgDhFL0aHcRgFqxO7AwQTtGr0UEMZsHqxM4A4RS9Gh3EYBasTuwMEE7Rq9FBDGbB6sTOAOEUvRodxGAWrE7sDBBO0avRQQxmwerEzgDhFL0aHcRgFqxO7AwQTtGr0UEMZsHqxM4A4RS9Gh3EYBasTuwMEE7Rq9FBDGbB6sTOAOEUvdqnDm5r3ijQmwWrEzsDhFPxanQQg1mwOrEzQDgVr0YHMZLVqh87A4RT8Wp0ECNZrfqxM0A4Fa9GBzGS1aofOwOEU/FqdBAjWa36sTNAOBWvRgcxktWqHzsDhFPxanQQI1mt+rEzQDgVr/a1g9ualwr0Y7Xqx84A4ZS7Gh3EMJaqruwMEE65q9FBDGOp6srOAOGUuxodxDCWqq7sDBBOuavRQQxjqerKzgDhlLsaHcQwlqqu7AwQTrmrNR3c1rxX4HTWqd7sDBBOravRQYxhnerNzgDh1LoaHcQY1qne7AwQTq2r0UGMYZ3qzc4A4dS62ncd3Na8WuBEFqkL2BkgnEJXo4MYwCJ1ATsDhFPoanQQA1ikLmBngHAKXY0OYgCL1AXsDBBOoas96eC25u0Cp7BCXcPOAOFUuRodxNWsUNewM0A4Va5GB3E1K9Q17AwQTpWrPe/gtuYFA2+yPF3GzgDhlLgaHcSlLE+XsTNAOCWu9mMHtzXvGHiZtelKdgYIp77V6CCuY226kp0BwqlvNTqI61ibrmRngHDqW+23Dm5rXjPwAgvTxewMEE5xq9FBXMTCdDE7A4RT3Go7OritedPAURami9kZIJzKVqODuIJV6Xp2BginstX2dXBb87KB/axK17MzQDhlrUYH0Z0laQg7A4RT1mp0EN1ZkoawM0A4Za22u4PbmvcN/Mp6NIqdAcKpaTU6iL6sR6PYGSCcmlY70sFtzSsHfmAxGsjOAOEUtBodREcWo4HsDBBOQasd7OC25q0D37ISjWVngHCqWe14B7c1Lx4wlqHh7AwQTimr0UF0YRkazs4A4ZSy2ksd3Na8e+CDNWgGdgYIp47V6CDOZw2agZ0BwqljtVc7uK15/cDGAjQJOwOEU8Rqb3RwW3MDCGf1mYedAcKpYDU6iDNZfeZhZ4BwKljtvQ5uay4BsSw9U7EzQDjlq/Z2B7c194BA1p3Z2BkgnNpVo4M4h3VnNnYGCKd21c7o4LbmKhDFojMhOwOEU7hqJ3VwW3MbCGHFmZOdAcKpWrXzOrituRDcnuVmWnYGCKdk1U7t4LbmTnBvlptp2RkgnHpVo4N4nbVmZnYGCKde1c7u4LbmWnBLFprJ2RkgnGJV69DBbc3N4GasMvOzM0A4larWp4PbmsvBbVhilmBngHDKVK1bB7c194MbsL6sws4A4dSoWs8ObmuuCEuzuCzEzgDhFKha5w5ua24Ji7KyrMXOAOFUp1r/Dm5rLgrLsawsx84A4ZSm2iUd3NbcFdZiWVmOnQHCqUu1qzq4rTktLMGCsig7A4RTlGoXdnBbc2OYnNVkXXYGCKci1a7t4Lbm0jAtS8nS7AwQTjmqXd7Bbc29YULWkdXZGSCcWlQb0cFtzdVhKhaRG7AzQDiFqDaog9ua28MkrCD3YGeAcKpQbVwHtzUXiOEsH7dhZ4BwSlBtaAcfa04RQ1g4bsbOAOEUn9oEHdzW3CQuZtW4HzsDhFN5anN0cFtzmbiMJeOW7AwQTtmpTdPBx5oTRVcWixuzM0A4Bac2WQe3NbeKTqwU92ZngHCqTW2+Dj7WHC3OZZm4PTsDhFNnarN2cFtzujiFBSKEnQHCKTK1iTv4WHPGeJmlIYqdAcIpL7XpO/hYc9I4xKIQyM4A4RSW2iId3NbcNnayImSyM0A4VaW2Tgcfa44cP7AWJLMzQDj1pLZaBx9rDh7GKgA7A4RTSWprdvCx5vixsfvH7ekc2BtbuYOPNSGIZeeBHLoF9urW7+DHmi7ksKtAIF0Be2k36uBjTSNuzC4ByfT+2Uu7XQc/1lTjNuwAgAe9fHZ89+3gx5qOLMoePdDSm2cHF9DBjzVlWYI9dOAHeurs4JI6+HlNbqZijxvYTy+cHVlqBz+vydAQ9pqBl+lhs92jg82aQnVibxc4i14y2z06uG9NxQ6xZwr0pnfL9o0OnjZ7iMBYepdsx+jgObMnCAynp8l2jA6eMHt/wCT0QNlvo4MnzB4fMAk9UPbb6OC7s5cHTEXPlP04OvjW7M0BE9JjZc9HB9+aPThgQnqs7Pno4Ouz1wZMS0+WPRkdfHH2zoDJ6eGy70YHX5w9MmByerjsu9HBV2YvDFiCni9rRgcPz94WsBA9YvZ1dPDw7GEBC9EjZl9HB4/NXhWwHD1l9ml08NjsSQEr0mtmNTp4YPaYgEXpQbMaHdw7e0nA0vSs2Z/Rwb2zZwQsTc+a/Rkd3Dt7RsDS9KzZn9HBA7OXBCxKD5rV6CBjLH10kDGWPjrIGEsfHWSMZe/vv/8PerrxQqMWmkAAAAAASUVORK5CYII=';
```

Comment:
I think the new image is better from the standpoint of not requiring CELA or similar approval, but I do think we should have an image that it is more refined/professional. Let me ask Daniel to help you find a great one.